layers/+distribution/spacemacs-base/packages.el

```diff
@@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
         i.e. `SPC m h h`, to lookup the source of the definition,
         while falling back to `evil-lookup'"
         (interactive)
-        (condition-case nil
+        (condition-case-unless-debug nil
```

Comment:
I agree :+1:

> My perspective is that you enable debugging because you want a backtrace and I'd just assume always generate one.

Coming from a Python perspective, errors are frequently used as part of an interface. That seems to be the case here, even though as you say it should have been written differently to begin with.

IOW, *errors* are not always *exceptions*.